modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/sidebar/MultiPanelSidebar.es.js

```diff
@@ -83,7 +81,8 @@ export default function MultiPanelSidebar({
 				setPanelComponents(result);
 			}
 		});
-	}, [isMounted, dispatch, load]);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why We are disabling this rule here?

Reply:
mmmmmmmm, this is weird because [`dispatch` is defined on the Context](https://github.com/liferay-forms/liferay-portal/blob/603bb3678628982f668fc8f3ec8cad0c83e4b7da/modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/App.es.js#L50) and I think We can rely this function [is already "callbacked"](https://pt-br.reactjs.org/docs/hooks-reference.html#usecallback) because it's provided by React itself and it's not losing the reference. Maybe it has been caused because [We are creating a new object](https://github.com/liferay-forms/liferay-portal/blob/603bb3678628982f668fc8f3ec8cad0c83e4b7da/modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/App.es.js#L87)? @matuzalemsteles, What do you think about it?